src/categorizer.py

```
@@ -26,146 +28,668 @@ def __init__(
+            "jitter": True,  # Adiciona aleatoriedade para evitar thundering herd
+            "retry_on_errors": [
+                "API_QUOTA_EXCEEDED",
+                "RATE_LIMIT_EXCEEDED", 
+                "INTERNAL_ERROR",
+                "TIMEOUT",
+                "CONNECTION_ERROR",
+                "JSON_DECODE_ERROR"
+            ]
+        }
+        
+        print(f"🔄 Configurando estratégia de retry:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)